ovs/lib/disk.py

```diff
@@ -49,7 +51,9 @@ def sync_with_reality(storagerouter_guid):
             DiskController._logger.exception('Could not connect to StorageRouter {0}'.format(storagerouter.ip))
             raise
 
-        disks, name_alias_mapping = DiskTools.model_devices(client)
+        node_id = System.get_my_machine_id()
```

Comment:
The node id is not of this machine but of the machine to sync the disks for
Storagerouter(storagerouter_guid).machine_id